base/array.jl

```diff
@@ -1987,3 +1987,139 @@ julia> symdiff([1,2,3],[3,4,5],[4,5,6])
 ```
 """
 symdiff(a, b, rest...) = symdiff(a, symdiff(b, rest...))
+
+## replace/replace! ##
+
+# NOTE: to implement replace! and replace for a new type T, it's enough to define:
```

Comment:
I'd rather not have this at all, AFAIK we don't generally do this, and people are not supposed to read this file to find out...